modules/sdk-coin-avaxp/src/lib/exportInCTxBuilder.ts

```diff
@@ -0,0 +1,210 @@
+    return this;
+  }
+
+  /**
+   * Set the nonce of C-Chain sender address
+   *
+   * @param {number | string} nonce - number that can be only used once
+   */
+  nonce(nonce: number | string): this {
+    const nonceBN = BN.isBN(nonce) ? nonce : new BN(nonce);
+    this.validateAmount(nonceBN);
+    this._nonce = nonceBN;
+    return this;
+  }
+
+  /**
+   * Export tx target P wallet.
+   *
+   * @param pAddresses
+   */
+  to(pAddresses: string | string[]): this {
+    const pubKeys = pAddresses instanceof Array ? pAddresses : pAddresses.split('~');
+    this.transaction._to = pubKeys.map(utils.parseAddress);
+    return this;
+  }
+
+  protected get transactionType(): TransactionType {
+    return TransactionType.Export;
+  }
+
+  initBuilder(tx: Tx): this {
+    const baseTx: BaseTx = tx.getUnsignedTx().getTransaction();
+    if (
+      baseTx.getNetworkID() !== this.transaction._networkID ||
+      !baseTx.getBlockchainID().equals(this.transaction._blockchainID)
+    ) {
+      throw new Error('Network or blockchain is not equals');
+    }
+
+    if (!this.verifyTxType(baseTx)) {
+      throw new NotSupported('Transaction cannot be parsed or has an unsupported transaction type');
+    }
+
+    // The outputs is a multisign P-Chain address result.
+    // It's expected to have only one outputs to the destination P-Chain address.
+    const outputs = baseTx.getExportedOutputs();
+    if (outputs.length !== 1) {
+      throw new BuildTransactionError('Transaction can have one output');
+    }
+    const output = outputs[0];
+
+    if (!output.getAssetID().equals(this.transaction._assetId)) {
+      throw new Error('AssetID are not equals');
+    }
+
+    // The inputs is not an utxo.
+    // It's expected to have only one input form C-Chain address.
+    const inputs = baseTx.getInputs();
+    if (inputs.length !== 1) {
+      throw new BuildTransactionError('Transaction can have one output');
+    }
+    const input = inputs[0];
+
+    this.transaction._to = output.getOutput().getAddresses();
+    const inputAmount = new BN((input as any).amount);
+    const outputAmount = (output.getOutput() as AmountOutput).getAmount();
+    const fee = inputAmount.sub(outputAmount);
+    this._amount = outputAmount.subn(Number(this.fixedFee));
+    this.transaction._fee.feeRate = fee.toNumber();
+    this.transaction._fee.fee = this.transaction._fee.feeRate.toString();
+    this.transaction._fee.size = 1;
+    this.transaction._fromAddresses = [input.getAddress()];
+
+    this._nonce = new BN((input as any).nonce);
+    this.transaction.setTransaction(tx);
+    return this;
+  }
+
+  static verifyTxType(baseTx: BaseTx): baseTx is ExportTx {
+    return baseTx.getTypeID() === EVMConstants.EXPORTTX;
+  }
+
+  verifyTxType(baseTx: BaseTx): baseTx is ExportTx {
+    return ExportInCTxBuilder.verifyTxType(baseTx);
+  }
+
+  /**
+   * Build the export in C-chain transaction
+   * @protected
+   */
+  protected buildAvaxpTransaction(): void {
```

Comment:
Shouldn't this be `buildAvaxpTransaction`?

Reply:
It renames from `buildAvaxpTransaction` to `buildAvaxTransaction`